configure.in

```diff
@@ -226,6 +226,7 @@ case "${host_os}" in
                  mac_version_min="-mmacosx-version-min=10.3"
                  mac_sysroot="-isysroot /Developer/SDKs/MacOSX10.4u.sdk"
               fi
+              mac_arches="x86_64"
               ;;
```

Comment:
It looks like this will set mac_arches to only the string "x86_64" but the code below will set the string to "-arch x86_64" is that the desired behavior?

Reply:
Good catch. Will fix.

Reply:
Done